test/client.test.js

```diff
@@ -386,12 +405,11 @@ describe('Client', () => {
       requiredKeys.every(key => driverEnvHeader.includes(key))
     ).toBeDefined()
   })
-
 })
 
-function assertHeader(headers, name) {
-  expect(headers[name]).not.toBeNull()
-  expect(parseInt(headers[name])).toBeGreaterThanOrEqual(0)
+function assertMetric(metrics, name) {
+  expect(metrics[name]).not.toBeNull()
+  expect(parseInt(metrics[name])).toBeGreaterThanOrEqual(0)
 }
```

Comment:
I notice here you have to convert the metric to an int.

It would be a better UX if we did this in the driver code itself - otherwise every client is going to have to do this conversion.

Reply:
Good call